devito/archinfo.py

```diff
@@ -71,6 +72,9 @@ def get_cpu_brand():
         cpu_info['flags'] = ci.get('flags')
         cpu_info['brand'] = ci.get('brand')
 
+        if cpu_info['brand'] is None:
```

Comment:
Can you add a comment about where (what platform) we saw "this" happening , ie the flags!=None while brand still None.

Reply:
ok